scripts/curl

```diff
@@ -84,6 +84,21 @@ for (const [re, fn] of /** @type {const} */([
             return undefined;
         }
     }],
+    [/^accounts\/((0x[0-9a-fA-F]{40})|(0\.0\.\d+))\/tokens/, idOrAliasOrEvmAddress => {
```

Comment:
I think we should have tests that ensure these regexs work and catch regressions.
Not an item for this PR as I see this file has the same issue in multiple places.
Could you create a follow up ticket to capture this ask

Reply:
This scripts is used for testing (to provide a drop-in replacement for `curl` while running tests to avoid hitting the real Mirror Node). Do we still want to create tests for this?

Reply:
+1, we use this `curl` override script only for the tests